.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
this limitation isn't necessary, as it prevents PRs targeting other branches from running CI

Reply:
Do you know why we have that limitation in spark-evaluate config?

https://github.com/filecoin-station/spark-evaluate/blob/f87560ec447d41ecaaadd3eafc7ab33d8eb2e078/.github/workflows/ci.yml#L5-L6

But not in spark-api config:

https://github.com/filecoin-station/spark-api/blob/0d79f8b060aac01d2f8c04d622aed3aded3dc707/.github/workflows/ci.yml#L5-L6

Git blame pointed me to this PR:

https://github.com/filecoin-station/spark-evaluate/pull/198

Reply:
https://github.com/filecoin-station/piece-indexer/pull/8